src/schemathesis/runner/checks.py

```diff
@@ -60,8 +64,12 @@ def response_schema_conformance(response: requests.Response, result: TestResult)
     schema = definition.get("schema")
     if not schema:
         return
+    if isinstance(response, requests.Response):
```

Comment:
Maybe it will be better to provide some kind of abstraction so code in checks will be unified